src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java

```diff
@@ -74,7 +79,7 @@ public BulkPrintService(
         backoff = @Backoff(delay = 200)
     )
     @Override
-    public void print(Claim claim, List<Printable> documents) {
+    public void print(Claim claim, List<Printable> documents, String authorisation) {
```

Comment:
You'll need to add `String authorisation` to the corresponding `@Recover` method so that they're linked by Spring

Reply:
thanks for checking this. Done.